webapp/src/app/core/IssueCard/issue-card.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="border border-border bg-card rounded-lg p-4 w-72 cursor-pointer" (click)="openIssue()">
+  <div class="flex justify-between items-center mb-2 text-xs text-gray-500">
+    <span class="font-bold flex justify-center items-center space-x-1">
+      @if (state() === 'OPEN') {
+        <ng-icon [svg]="octGitPullRequest" size="16" class="mr-1 text-github-success-foreground"></ng-icon>
+      } @else {
+        <ng-icon [svg]="octGitPullRequestClosed" size="16" class="mr-1 text-github-danger-foreground"></ng-icon>
+      }
+
+      {{ repositoryName() }} #{{ number() }} on {{ createdAt().format('MMM D') }}
+    </span>
+    <span class="flex items-center space-x-2">
+      <span class="text-github-success-foreground font-bold">+{{ additions() }}</span>
+      <span class="text-github-danger-foreground font-bold">-{{ deletions() }}</span>
+    </span>
+  </div>
+
+  <div class="flex justify-between font-bold text-sm mb-3">
+    {{ title() }}
+    @if (getMostRecentReview(); as review) {
+      @if (review.state === 'APPROVED') {
+        <ng-icon [svg]="octCheck" size="16" class="text-github-success-foreground"></ng-icon>
+      } @else if (review.state === 'DISMISSED') {
+        <ng-icon [svg]="octX" size="16" class="text-github-danger-foreground"></ng-icon>
+      } @else if (review.state === 'COMMENTED') {
+        <ng-icon [svg]="octComment" size="16" class="text-github-muted-foreground"></ng-icon>
+      } @else {
+        <ng-icon [svg]="octFileDiff" size="16" class="text-github-danger-foreground"></ng-icon>
+      }
+    }
+  </div>
+  <div class="flex gap-1 flex-wrap">
+    @for (label of pullRequestLabels(); track label.name) {
+      <span class="px-2 py-1 rounded-full text-xs mr-2 text-white" [style.background-color]="label.color">{{ label.name }}</span>
```

Comment:
Github uses CSS's `calc()`, which I'm not familiar with either. But we could reduce the contrast with something like this:
```suggestion
      <span class="px-2 py-1 rounded-full text-xs mr-2 text-neutral-100/95" [style.background-color]="label.color">{{ label.name }}</span>
```